rclpy/rclpy/qos.py

```diff
@@ -270,8 +270,11 @@ class QoSHistoryPolicy(QoSPolicyEnum):
     RMW_QOS_POLICY_HISTORY_KEEP_ALL = 2
     KEEP_ALL = RMW_QOS_POLICY_HISTORY_KEEP_ALL
 
+#  Naming the class HistoryPolicy QoSHistoryPolicy 
+#  because it is used in other places with the "Qos" infront
```

Comment:
Maybe, something like: `Alias with the old name, for retrocompatibility` is clearer.